code/__HELPERS/roundend.dm

```diff
@@ -102,6 +102,14 @@
 
 	CHECK_TICK
 
+	// Add AntagHUD to everyone, see who was really evil the whole time!
+	for(var/datum/atom_hud/H in GLOB.huds)
```

Comment:
![](https://file.house/z0e_.png)

Reply:
literally what is done in the AntagHUD admin verb

Reply:
ah well, I might look at keying this list at some point